src/Classifiers/ResourceClassifier.php

```diff
@@ -17,10 +16,6 @@ public function name(): string
 
     public function satisfies(ReflectionClass $class): bool
     {
-        if ($class->isSubclassOf(Resource::class)) {
-            return true;
-        }
-
```

Comment:
The `Illuminate\Http\Resources\Json\Resource` class has been removed in Laravel 7.0.
As we no longer support Laravel versions lower than 7.0, we no longer have to keep this check here.